crates/ruff_python_formatter/src/expression/expr_constant.rs

```diff
@@ -39,6 +39,22 @@ impl FormatNodeRule<ExprConstant> for FormatExprConstant {
             }
         }
     }
+
+    fn fmt_dangling_comments(
+        &self,
+        _node: &ExprConstant,
+        _f: &mut PyFormatter,
+    ) -> FormatResult<()> {
+        // TODO(konstin): Reactivate when string formatting works, currently a source of unstable
+        // formatting, e.g.:
+        // magic_methods = (
+        //     "enter exit "
+        //     # we added divmod and rdivmod here instead of numerics
+        //     # because there is no idivmod
+        //     "divmod rdivmod neg pos abs invert "
```

Comment:
I've long wondered if it'd be a simplification to store implicit string concatenations in the AST (like explicit concatenations). The need to re-parse strings and deal with implicit concatenations bites us everywhere.

Reply:
yeah i'd prefer it if the AST would keep the string parts as separate